raiden/network/channelgraph.py

```diff
@@ -34,16 +55,87 @@ def make_graph(edge_list):
     return graph
 
 
+def route_to_routestate(route):
+    path = route.path
+    next_hop = path[1]
+    channel = route.channel
+
+    return channel_to_routestate(channel, next_hop)
+
+
+def channel_to_routestate(channel, node_address):
+    state = channel.state
+    chanel_address = channel.external_state.netting_channel.address
+    distributable = channel.distributable
+    settle_timeout = channel.settle_timeout
+    reveal_timeout = channel.reveal_timeout
+
+    if state == 'closed':
+        close_block = channel.external_state.closed_block
+    else:
+        close_block = None
+
+    state = RouteState(
+        state=state,
+        node_address=node_address,
+        chanel_address=chanel_address,
+        available_balance=distributable,
+        settle_timeout=settle_timeout,
+        reveal_timeout=reveal_timeout,
+        close_block=close_block,
+    )
+
+    return state
+
+
 class ChannelGraph(object):
     """ Has Graph based on the channels and can find path between participants. """
 
-    def __init__(self, edge_list):
-        """
-        Args:
-            edge_list (List[(address1, address2)]): all the netting contracts
-                that participate in the network.
+    def __init__(
+            self,
+            our_address_bin,
```

Comment:
I believe we agreed on dropping the `_bin` suffix and only annotate `_hex` where applicable.